crates/ruma-client-api/src/dehydrated_device.rs

```diff
@@ -46,6 +49,26 @@ impl DehydratedDeviceV1 {
     }
 }
 
+/// The `org.matrix.msc3814.v2` variant of a dehydrated device.
+#[derive(Clone, Debug)]
+#[cfg_attr(not(feature = "unstable-exhaustive-types"), non_exhaustive)]
```

Comment:
This is not done with a feature anymore but with a cfg attribute:

```suggestion
#[cfg_attr(not(ruma_unstable_exhaustive_types), non_exhaustive)]
```